routes/middleware.js

```diff
@@ -20,9 +20,7 @@ var _ = require('lodash');
 exports.initLocals = function (req, res, next) {
 	res.locals.navLinks = [
 		{ label: 'Home', key: 'home', href: '/' },
-		{ label: 'Laws', key: 'laws', href: '/laws' },
-		{ label: 'Explanations', key: 'explanations', href: '/explanations' },
-		{ label: 'Talk to an Expert', key: 'talk-to-an-expert', href: '/talk-to-an-expert' },
+		{ label: 'Blog', key: 'blog', href: '/blog' },
```

Comment:
why?

Reply:
this shouldn't be there. I intended the opposite.

Reply:
resolved. only `talk-to-an-expert` is the missing navLink